src/System.Memory/src/System.Memory.csproj

```diff
@@ -11,12 +11,12 @@
 
   <!-- Package servicing properties -->
   <PropertyGroup>
-    <IsPackable>true</IsPackable>
-    <VersionPrefix>4.6.1</VersionPrefix>
-    <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.6.2</VersionPrefix>
-    <AssemblyVersion Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.0.3.0</AssemblyVersion>
-    <AssemblyVersion Condition="'$(IsPackable)' == 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.0.4.0</AssemblyVersion> <!-- Keep the new assembly version under 4.1.x.x! -->
-    <PackageValidationBaselineVersion>4.6.0</PackageValidationBaselineVersion>
```

Comment:
I can't find previous comments to confirm this, but did I miss bumping this in previous PRs, or is this baseline expected to always be 4.6.0?